### PR TITLE
pacmap of progen

### PR DESCRIPTION
**Pull request by andrewram4287**
@aarmey Currently its hard to visualize the cells in the pacmap

### CODE REVIEW

File: sccp/figures/figureLupus16.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+def plotPartialCmpUMAP(X, cmp: int, ax: Axes):
+    """Scatterplot of UMAP visualization weighted by
+    projections for a component and cell state"""
+    weightedProjs = X.obsm["weighted_projections"]
+    weightedProjs = weightedProjs[:, cmp - 1]
+    weightedProjs = weightedProjs / np.max(np.abs(weightedProjs)) * 2.0
+
+    cmap = sns.diverging_palette(240, 10, as_cmap=True, s=100)
+    plot = points(X.obsm["embedding"], values=weightedProjs, cmap=cmap, ax=ax)
```
Comment (aarmey) [inline sccp/figures/figureLupus16.py:42]: Try using a standard matplotlib/seaborn plot here. There aren't a huge number of cells, so it should work fine.

File: sccp/figures/figureLupus16.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+
+def plotPartialCmpUMAP(X, cmp: int, ax: Axes):
+    """Scatterplot of UMAP visualization weighted by
+    projections for a component and cell state"""
+    weightedProjs = X.obsm["weighted_projections"]
+    weightedProjs = weightedProjs[:, cmp - 1]
+    weightedProjs = weightedProjs / np.max(np.abs(weightedProjs)) * 2.0
+
+    cmap = sns.diverging_palette(240, 10, as_cmap=True, s=100)
```
Comment (aarmey) [inline sccp/figures/figureLupus16.py:41]: This colormap is white at 0, where most of your cells are. This is definitely part of the problem.

**Review by aarmey — COMMENTED** [review]
@andrewram4287 two suggestions here.

**Comment by andrewram4287** [comment]
@aarmey Thanks for the suggestions!

**Pull request closed without merging**